src/main/java/com/resources/controller/ControllerAplication.java

```diff
@@ -1,16 +1,50 @@
 package com.resources.controller;
 
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.MongoDatabaseFactory;
+import org.springframework.data.mongodb.core.convert.DbRefResolver;
+import org.springframework.data.mongodb.core.convert.DefaultDbRefResolver;
+import org.springframework.data.mongodb.core.convert.DefaultMongoTypeMapper;
+import org.springframework.data.mongodb.core.convert.MappingMongoConverter;
+import org.springframework.data.mongodb.core.mapping.MongoMappingContext;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.client.RestTemplate;
+import com.resources.model.User;
+import com.resources.model.UserId;
+import com.resources.services.ServiceApplication;
 
+@Configuration
 @RestController
 public class ControllerAplication {
 	RestTemplate restTemplate; 
 	
-	@RequestMapping(value = "/")
-	public String main() {
-	     System.err.println("Hola");
-	      return "Hello at-sso-api";
-	   }
+	@Autowired
+	private MongoDatabaseFactory mongoDatabaseFactory;
+	
+	@Autowired
+	private MongoMappingContext mongoMappingContext;
+	
+	@Autowired
+	private ServiceApplication serviceApplication;
+	
+	@Bean
+	public MappingMongoConverter mappingMongoConverter() {
+		DbRefResolver dbRefResolver = new DefaultDbRefResolver(mongoDatabaseFactory);
+		MappingMongoConverter converter = new MappingMongoConverter(dbRefResolver, mongoMappingContext);
+		converter.setTypeMapper(new DefaultMongoTypeMapper(null));
+		return converter;
+	}
+	
+	@PostMapping(value = "/at-sso-api/api/v1/user", produces = "application/json")
+	@ResponseStatus(value = HttpStatus.CREATED)
+	public UserId createUser(@RequestBody User user) {
+		UserId userId = serviceApplication.createUser(user);
+		return userId;
```

Comment:
just be sure that we returning the right Response:

You can use `return new ResponseEntity<>(_userObject, HttpStatus.CREATED);`